packages/ui/src/multiplying-architecture/KaotoBridge.tsx

```diff
@@ -45,18 +46,15 @@ interface KaotoBridgeProps {
    * ChannelType where the component is running.
    */
   channelType: ChannelType;
-
-  /**
-   * Catalog URL to load the components and schemas.
-   */
-  catalogUrl: string;
```

Comment:
There's no need to receive the `catalogUrl` as a prop since we're using the `SettingsAdapter`